docs/man/xrdp.ini.5.in

```diff
@@ -115,8 +111,11 @@ Specify text passed to PAM when authentication failed. The maximum length is \fB
 
 .TP
 \fBport\fP=\fIport\fP
-Specify TCP port to listen on for incoming connections.
-The default for RDP is \fB3389\fP.
+Specify TCP port and interface to listen on for incoming connections.
+Specifying only the port means that xrdp will listen on all interfaces.
+The default port for RDP is \fB3389\fP.
+Multiple address:port instances must be separated by spaces. Check the .ini file for examples.
```

Comment:
This is correct but actually commas `,` are also supported to separate instances.

Reply:
Updated the PR